scripts/install-dependencies.sh

```
@@ -0,0 +1,56 @@
+echo "🏠 Installing dependencies in root folder..."
+cd "$PROJECT_ROOT"
+if npm install --legacy-peer-deps; then
+    echo "✅ Root dependencies installed successfully"
+else
+    echo "❌ Failed to install root dependencies"
+    exit 1
+fi
+
+# Install dependencies in example folder
+echo "📱 Installing dependencies in example folder..."
+cd "$PROJECT_ROOT/example"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)